main.py

```diff
@@ -0,0 +1,147 @@
+import time
+from user import *
+from csv_utility import *
+from random_id_generator import *
+from chirps import *
+from chirps_utility import *
+from user_utility import *
+from convo_utility import *
+
+class MainMenu:
+
+	def __init__(self):
+		print("")
+		print(" ~ WELCOME TO BIRDYBOARD ~ ")
+		self.users = {}
+		self.chirps = {}
+		self.current_user = {}
```

Comment:
Since `self.current_user` isn't _ever_ a dictionary, it might make more sense to initialize it as `None`.